source/material_model/visco_plastic.cc

```diff
@@ -730,7 +860,13 @@ namespace aspect
     ViscoPlastic<dim>::parse_parameters (ParameterHandler &prm)
     {
       // increment by one for background:
-      const unsigned int n_fields = this->n_compositional_fields() + 1;
+      unsigned int n_fields = 0;
```

Comment:
Why does the above not work? In theory you should have access to that variable already.